YoutubeExplode/Bridge/YoutubeController.cs

```diff
@@ -124,6 +124,23 @@ internal class YoutubeController
             );
         }
 
+        public async Task<PlayerResponseExtractor?> GetPlayerResponseFromEndpoint(VideoId videoId)
+        {
+            string url = "https://www.youtube.com/youtubei/v1/player";
+
+            //Use this as mentioned on https://github.com/Tyrrrz/YoutubeExplode/issues/581#issuecomment-889241520
+            string requestJson = $@"{{""context"": {{""client"": {{""clientName"": ""ANDROID"",""clientScreen"": ""EMBED"",""clientVersion"": ""16.05""}},""thirdParty"": {{""embedUrl"": ""https://www.youtube.com""}}}},""videoId"": ""{videoId}"" }}";
+
+
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Post,url);
+            httpRequestMessage.Headers.Add("X-Goog-Api-Key", InternalApiKey);
+            httpRequestMessage.Content = new StringContent(requestJson, System.Text.Encoding.UTF8, "application/json");
```

Comment:
Can we rewrite it to use object serialization, similar to `GetPlaylistAsync(...)` method? The string is a bit hard to read.

https://github.com/Tyrrrz/YoutubeExplode/blob/314be1cade649064ba8f3e6620b812a4ea651539/YoutubeExplode/Bridge/YoutubeController.cs#L160-L187